### DISCUSSION TITLE
posts/623

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#6] [Announcements]
# posts/623

안녕하세요. 맵비전플랫폼개발의 Zayden(이동진)입니다.
제가 속한 조직은 카...

https://tech.kakao.com/posts/623

**Comment by jws1837** [comment]
도움많이 됐습니다!

**Comment by kimyooonkyoung** [comment]
👏🏻👏🏻👏🏻감사합니다😄

**Comment by ujinyang** [comment]
최근 이미지모델간 성능테스트 했을 때, 여전히 트랜스포머기반 모델의 성능이 뛰어났었습니다.
EMA나 OOD를 실제하고 학습이미지 선분류내용 또한 많은 도움이 되었습니다.
내용에 없는 이미지 증강기법 부분의 내용도 궁금합니다.
